.github/workflows/run-checks-and-tests.yaml

```
@@ -0,0 +1,70 @@
+
+  run-e2e-tests:
+    name: Run e2e tests
+    needs:
+      - run-code-quality-checks
+    permissions:
+      contents: read
+    uses: ./.github/workflows/run-e2e-tests.yaml
+
+  run-graphql-fuzz-tests:
+    name: Run GraphQL fuzz tests
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)